### PR TITLE
Added bower.json file

### LINKED ISSUE #1: Add bower.json file
Hi!

Just wanted to know if it could be possible to add a `bower.json` file, then adding `ignore` rules to it.

This way, it will be possible to fetch only the JS file without getting tests, HTML files, etc.

What do you think @arv?

### PR DESCRIPTION
**Pull request by Ninir**
Closes #1

### CODE REVIEW

**Comment by birbilis** [comment]
"The project moved to google code" - so since that one is closing where did it go from there?

**Comment by arv** [comment]
It moved from SourceForge to Google Code to GitHub. Not moving again.

**Comment by birbilis** [comment]
GitHub and Google Code have done the super silly thing to allow you to export a project from Google Code into GitHub directly, instead of having you fork from an immutable (till claimed by the original author) repository that would be autogenerated at first demand on GitHub. That way there are several ExplorerCanvas now on GitHub, but not sure which is the main one, plus you don't see them as related in GitHub to follow their tracks and merge from one to another etc. What a shame

**Comment by arv** [comment]
@birbilis Are you sure that these are just not forks of this repo? If you click on the Fork number in the upper right it will show you the network and how the forks are related.

**Comment by birbilis** [comment]
See webjars and cowlibob versions for example, maybe more there. Not sure if they were created via google code export, but most probably that's the case

Google code repos don't point to GitHub version unless edited (before freezing?) to have such text. So user thinks latest version is there and try to clone repository, but they're instead offered to export to GitHub where their project becomes the parent one, unrelated to exports from others

**Comment by birbilis** [comment]
See https://github.com/cowlibob/explorercanvas
refers to google code version

(webjars one might be something like a wrapper, or unrelated, but not sure)

Google should allow project author to disable that feature in my opinion or generate a master project (frozen) on GitHub and fork from that. When an owner of the project exported themselves later (if they ever did) it would give it to them, although that could be optional and could have all autofork from immutable exported projects under a googlecode account at GitHub

File: bower.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        "Google Open Source",
+        "Emil A Eklund",
+        "Erik Arvidsson",
+        "Glen Murphy"
+    ],
+    "keywords": [
+        "javascript",
+        "canvas"
+    ],
+    "description": "Modern browsers like Firefox, Safari, Chrome and Opera support the HTML5 canvas tag to allow 2D command-based drawing. ExplorerCanvas brings the same functionality to Internet Explorer. To use, web developers only need to include a single script tag in their existing web pages.",
```
[the remark this answers is not in the document]
Comment (nigon135) [reply bower.json]: Good idea 👍, done :)

**Comment by nigon135** [comment]
on pull request

**Comment by birbilis** [comment]
I see a "This branch has conflicts that must be resolved" here

**Comment by Ninir** [comment]
This was done in https://github.com/arv/ExplorerCanvas/commit/b2a7c3b4fc191924404ddc5bf5036c6a58b18542

**Pull request closed without merging**